resources/conf/config.toml.template

```diff
@@ -364,6 +364,15 @@ idleTimeoutInSeconds = 3600
   [controlPlane.jmsConnectionParameters]
     eventListeningEndpoints = ["amqp://admin:$env{cp_admin_pwd}@apim:5672?retries='10'&connectdelay='30'"]
 
+# Global Adapter related configurations
+[globalAdapter]
+  # Enable Choreo Connect Global adapter client
+  enabled = false
+  # Global adapter Service URL (<Hostname>:<Port>)
+  serviceUrl = "ga:18002"
+  # Local Adapter Label
+  localLabel = "$env{LocalLabel}"
```

Comment:
Shall we keep env variable names to a constant pattern like we have used in this file.
Either **local_label** or **LOCAL_LABEL**

Reply:
fixed using local_label